core/common/src/main/java/alluxio/network/protocol/RPCBlockReadResponse.java

```diff
@@ -24,6 +24,8 @@
 
 /**
  * This represents the response of a {@link RPCBlockReadRequest}.
+ *
+ * Since 1.4.1, this also represents the response of a {@link RPCUnderFileSystemBlockReadRequest}.
```

Comment:
done. I was not sure about the plan. i do not think this should be in a patch release either.